### PR TITLE
[BPK-3277] Add script for downloading fonts

### PR DESCRIPTION
**Pull request by georgegillams**
+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-ios/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] Adding a component? Remember to expose it in the [main `Backpack.h` header file](https://github.com/Skyscanner/backpack-ios/tree/master/Backpack/Backpack.h)
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: Example/Backpack/BPKAppDelegate.m [hunk 1]
```diff
@@ -38,6 +39,8 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
             @{NSForegroundColorAttributeName: BPKColor.textPrimaryColor};
     }
 
+//    [BPKFont setFontDefinition:[BPKRelativeFontDefinition new]];
```
Comment (imchristian) [inline Example/Backpack/BPKAppDelegate.m]: 🚨

File: scripts/download-relative-fonts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+mkdir bpk-fonts-tmp
```
Comment (imchristian) [inline scripts/download-relative-fonts]: Can this script be executed automatically after the `bundle exec pod update`? https://guides.cocoapods.org/syntax/podfile.html#post_install

File: Example/Backpack/BPKAppDelegate.m (hunk 1 above)
Comment (georgegillams) [reply Example/Backpack/BPKAppDelegate.m]: This gets automatically un-commented by the script

File: scripts/download-relative-fonts (hunk 2 above)
Comment (georgegillams) [reply scripts/download-relative-fonts]: Yeah - although we'd need to ensure it fails gracefully when SSH access to GitHub fails

File: Example/Backpack/BPKAppDelegate.m (hunk 1 above)
Comment (imchristian) [reply Example/Backpack/BPKAppDelegate.m]: giphy-4

**Pull request merged**